File: FBOLinx.Web/FBOLinx.Web/ClientApp/src/app/models/UserAcceptedPolicyAndAgreements.ts
import { PolicyAndAgreementDocuments } from "./PolicyAndAgreementDocuments";
import { User } from "./user";

export interface UserAcceptedPolicyAndAgreements {
    oid: number;
    userId: number;
    documentId: number;
    acceptedDateTime: string;
    documents: PolicyAndAgreementDocuments[];
    user: User;
}

File: FBOLinx.Web/FBOLinx.DB/Models/UserAcceptedPolicyAndAgreements.cs
using System;
using System.Collections.Generic;

namespace FBOLinx.DB.Models
{
    public class UserAcceptedPolicyAndAgreements : FBOLinxBaseEntityModel<int>
    {
        public int UserId { get; set; }
        public int DocumentId { get; set; }
        public DateTime AcceptedDateTime { get; set; }
        public ICollection<PolicyAndAgreementDocuments> Documents { get; set; }
        public User User { get; set; }
    }
}
